src/App.js

```diff
@@ -1,18 +1,46 @@
 import React, { Component } from 'react';
-import logo from './logo.svg';
+import AddTask from './AddTask';
+import TaskList from './TaskList';
 import './App.css';
 
 class App extends Component {
+  constructor(props) {
+    super(props);
+
+    this.state = {
+      inputValue: '',
+      tasks: []
+    };
+  }
+
+  handleChange(value) {
+    this.setState({ inputValue: value });
+  }
+
+  addTask() {
+    var tasks = this.state.tasks.slice();
```

Comment:
In react, we never want to [mutate state directly](https://reactjs.org/docs/state-and-lifecycle.html#using-state-correctly). We'll always use `setState()`, like below. Here we're making a shallow copy of the tasks array first using [.slice()](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/slice) and then we push the new task on the array and call `setState()`.